### DISCUSSION TITLE
Has anyone made a zepeto world yet

### DISCUSSION THREAD

**Discussion by ZepFireFox** [naverz/zepeto-studio-kor#159] [4. General]
Has anyone made a zepeto world yet

**Comment by x4cf** [comment]
hello :)

**Comment by carolandreazza** [comment]
Hello! Does anyone have any more detailed information on how to create a world in Zepeto? I'm building in Unity, but we don't have much information in the guide.

If anyone knows, please:
Is it possible to send my world created in Unity? How it works?
How are interactions with objects made? Example: sitting in a chair

**Reply by natural-jogi** [reply]
All tutorials and practices are written in Korean atm, and I'm not sure whether Naver Z would make an English tutorial for global developers tbh... There are Korean tutorials **[here](https://studio.zepeto.me/guides/creating-your-own-world)**, I recommend using Google translate or something to check out.

**Reply by x4cf** [reply]
even, this, helps, ?
https://www.youtube.com/watch?v=U9PNr2VNc34

**Reply by Zepeto-TechEvangelist** [reply]
Hello 
You can see the guide document translated into English by changing the language setting at the bottom of the guide document to English.
Thank you 🥰

<img width="1469" alt="스크린샷 2022-02-24 오후 3 41 00" src="https://user-images.githubusercontent.com/100118650/155472098-cc90ecad-9ec9-4d05-9dd5-ab34e7346f02.png">
.
